cli/cmd/testdata/inject-filepath/expected/injected_nginx.yaml

```diff
@@ -99,6 +103,12 @@ spec:
           value: linkerd-identity.$(_l5d_ns).serviceaccount.identity.$(_l5d_ns).$(_l5d_trustdomain)
         - name: LINKERD2_PROXY_DESTINATION_SVC_NAME
           value: linkerd-destination.$(_l5d_ns).serviceaccount.identity.$(_l5d_ns).$(_l5d_trustdomain)
+        - name: LINKERD2_PROXY_POLICY_SVC_ADDR
+          value: linkerd-policy.linkerd.svc.cluster.local.:8090
```

Comment:
The namespace here is rendered but not in the next env variable because we render them in different ways (weirdly) which is also the case with destination 🤔 

https://github.com/linkerd/linkerd2/blob/ac2ef49957c9621860f4525b1da539977678d743/cli/cmd/testdata/inject-filepath/expected/injected_nginx.yaml#L28

Reply:
yeah I noticed this. it would be nice to make them uniform in a separate change -- I don't know why we'd ever need `$(_l5d_ns)` or `$(_l5d_trustdomain)` to be in the environment -- they're known by the injector without needing the downward API?

Reply:
Yeah, Makes Sense! I will raise a separate PR to fix this!